tests/test_benchmarks/test___init__.py

```diff
@@ -107,13 +107,21 @@ def run_test(self, benchmark, file, expected):
         with open(precomputed_features, 'rb') as f:
             precomputed_features = pickle.load(f)['data']
         precomputed_features = precomputed_features.stack(presentation=['stimulus_path'])
+        precomputed_paths = set(map(lstrip_local, precomputed_features['stimulus_path'].values))
         # attach stimulus set meta
         stimulus_set = benchmark._assembly.stimulus_set
-        expected_stimulus_paths = [stimulus_set.get_image(image_id) for image_id in stimulus_set['image_id']]
-        assert (precomputed_features['stimulus_path'].values == expected_stimulus_paths).all()
+        expected_stimulus_paths = list(
+            map(lstrip_local, [stimulus_set.get_image(image_id) for image_id in stimulus_set['image_id']]))
+        assert (precomputed_paths == set(expected_stimulus_paths))
         for column in stimulus_set.columns:
             precomputed_features[column] = 'presentation', stimulus_set[column].values
         precomputed_features = PrecomputedFeatures(precomputed_features)
         # score
         score = benchmark(precomputed_features).raw
         assert score.sel(aggregation='center') == expected
+
+
+def lstrip_local(path):
+    parts = path.split(os.sep)
+    path = os.sep.join(parts[-3:])
```

Comment:
I think it would be better to keep this flexible with respect to the root directory (i.e. like model-tools is doing it for `~/.brainio` https://github.com/brain-score/model-tools/blob/b378404216bde499562f602670d929062b357605/model_tools/activations/core.py#L224). I'd just use the same function here.

Reply:
I tried that but it's a bit messy. The paths can be different after `/brainio` sometimes images are in another `data` folder and sometimes they are not. I could not really figure out which version is the actual correct one but either way I want the test to work if the correct images are available, so I rewrote the test this way. Additionally if I would use the model tools implementation I would have to add model tools as a dependency to brain-score which is not the case right now and I would like to leave it like that.